src/main/java/org/omnifaces/util/Utils.java

```diff
@@ -270,9 +271,10 @@ public static boolean isBlank(String string) {
     public static boolean isNumber(String string) {
         try {
             // Performance tests taught that this approach is in general faster than regex or char-by-char checking.
-            return Long.valueOf(string) != null;
+            Long.parseLong(string);
+            return true;
         }
-        catch (Exception ignore) {
+        catch (NumberFormatException ignore) {
```

Comment:
This doesn't cover NullPointerException.

Reply:
The `Long.parseLong`, internally cover all the situations and throws a `NumberFormatException`, ... but for uniformity I'll bring back the generic `Exception'